roles/chrony/tasks/configure.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Upload chrony.conf template
+  tags: [chrony, chrony-config]
+  template:
+    src: chrony.conf.j2
+    dest: "{{ chrony_dir }}{{ chrony_filename }}"
```

Comment:
Add a `/` between `chrony_dir` and `chrony_filename`. It should work well in both cases: if `chrony_dir` ends with `/` and if it doesn't. So it's a way to make the role less error-prone.

Reply:
Reading again my comment, I'm not sure if it's clear. So let me clarify. These paths have the same meaning:
~//.ssh
~/.ssh
Double slashes are interpreted as single slashes.